qa/L0_dyna_sequence_batcher/test.sh

```diff
@@ -69,11 +69,16 @@ rm -fr *.log *.serverlog
 
 # models
 rm -fr models && mkdir models
-cp -r ${DATADIR}/$MODEL_REPOSITORY/* models/.
+for MODEL in ${DATADIR}/$MODEL_REPOSITORY/* ; do
+    cp -r $MODEL models/. && \
+        (cd models/$(basename $MODEL) && \
+            sed -i "s/kind: KIND_CPU/kind: KIND_CPU\\ncount: 1/" config.pbtxt)
```

Comment:
I thought we only increased the default instance count to 2 for TRT, ONNX, and TF. Are we setting the default instance count to two for all the backends?

Reply:
We decided in the [PR](https://github.com/triton-inference-server/core/pull/82) to go for an opt in approach. The only backends which are opted in are tensorflow and onnxruntime.

Reply:
I see. I think I commented on the wrong place. I was wondering why we need to set instance count of 1 for custom backend models? Is it just to be safe?